projects/admin/src/app/record/detail-view/document-detail-view/holding/default-holding-item/default-holding-item.component.html

```diff
@@ -64,10 +64,24 @@
           <div class="col-4 pl-5 font-weight-bold label-title" translate>Online access</div>
           <div class="col-8"><a target="_blank" href="{{ item.metadata.url }}">{{ item.metadata.url }}</a></div>
         </ng-container>
+        <!-- TEMPORARY LOCATION -->
+        <ng-container *ngIf="item.metadata.temporary_location">
+          <div class="col-4 pl-5 font-weight-bold label-title" translate>Temporary location</div>
+          <div class="col-8">{{ item.metadata.temporary_location.pid | getRecord: 'locations' : 'field' : 'name' | async }}</div>
+        </ng-container>
+        <!-- ITEM IN COLLECTION -->
+        <ng-container *ngIf="item.metadata.pid | itemInCollection | async as collections">
+          <div class="col-4 pl-5 font-weight-bold label-title" translate>Exhibition/course</div>
+          <div class="col-8">
+            <ng-container *ngFor="let collection of collections; let last=last">
+              <a [routerLink]="['/records', 'collections', 'detail', collection.metadata.pid]">{{ collection.metadata.title }}</a>
+              {{ last ? '' : '; ' }}
+            </ng-container>
+          </div>
+        </ng-container>
```

Comment:
If you item has a `temporary_location` and belongs to one/more `collections`, you display both informations. Right ? 
@pronguen Is it the correct use-case ? I understood that we : 
1) display collection informations if item belong to a collection
2) **if not** and item has a `temporary_location`, then display this information.
3) **if not** display nothing

Reply:
@zannkukai Bertrand told me he implemented a "cascade" for the public view: so collection, else temporary_location, else nothing.
For the pro interface, we finally decided to display both with the real label.